### PR TITLE
Ridge multiple lags

### PR DESCRIPTION
**Pull request by jannesvaningen**
Added a lags for loop in the ridge ipynb to accomodate fitting a ridge model for multiple lags add once.
I also added a line plot to show the y_train and y_test outcomes of the model.

### CODE REVIEW

File: workflow/comp_pred_ridge_and_LSTM.ipynb [hunk 1]
```diff
@@ -34,23 +34,13 @@
   },
```
Comment (geek-yang) [inline workflow/comp_pred_ridge_and_LSTM.ipynb:34]: <pre class="ql-syntax" spellcheck="false">Line #8.        f"The MSE of climatology is {mean_squared_error(ground_truth, np.repeat(target_clim, ground_truth.anchor_year.size)):.3f}"
</pre><p>Thanks for correcting it. I forgot to update this with climatology. Btw, for a fair evaluation, I think when calculating the climatology, we shall only include the training data, instead of the whole dataset (if you are looking for something else, just ignore my comment).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

**Review by geek-yang — COMMENTED** [review]
Thanks a lot for adding the multi lag regression to our recipes for a fair comparison. I planned to do it but didn't manage to find time. 😉 

Just a small comment, it seems the deep learning ones are not very robust. Maybe we can add random seeds, to make the results reproducible, for instance:

```py
torch.manual_seed(1)
# for python related randomness
random.seed(1)
# for numpy related randomness
np.random.seed(1)
```
(more information https://pytorch.org/docs/stable/notes/randomness.html)

Nice work @jannesvaningen !

**Review by semvijverberg — APPROVED** [review]
I check all notebooks, fixed some merge issue related to later updates (fixed seed + downloading data). 

+Now computing mean_squared_error for every notebook in the same manner.
